src/DataDictionary/DataDictionaryNode.less

```diff
@@ -0,0 +1,40 @@
+.download-button:hover,
```

Comment:
I think these should be name `data-dictionary-node__download-button` since they are in the `DataDictionaryNode` component